src/main/java/edu/kis/powp/jobs2d/visitor/VisitorDriverOperation.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.jobs2d.visitor;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+public class VisitorDriverOperation implements IVisitor {
```

Comment:
It is not clear what we are visiting. The info should be in the name of the class and the interface